core-processor/src/executor.rs

```diff
@@ -56,45 +57,46 @@ fn make_checks_and_charge_gas_for_pages<'a>(
     }
 
     let mem_size = if let Some(max_wasm_page) = allocations.iter().next_back() {
```

Comment:
in fact, charging in executor without lazy pages goes already after page loading